docs/checks/edgeCrossingEdgeCheck.md

```diff
@@ -0,0 +1,25 @@
+# EdgeCrossingEdgeCheck
+
+#### Description
+
+The purpose of this check is to identify Edges intersecting another Edge(s) that do not share the same Node (meaning they are not well-connected) nor have proper layer tagging on one of these Edge(s) (meaning there should be a layer tag for one of the Edges).
+
+#### What is Not Supported
```

Comment:
Not sure we need this section... If anything, I would move it to the bottom of the file

Reply:
Cool, I'll omit this section.